deployments/liqo/templates/_helpers.tpl

```diff
@@ -0,0 +1,105 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "liqo.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "liqo.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "liqo.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "liqo.labels" -}}
+helm.sh/chart: {{ include "liqo.chart" . }}
+{{ include "liqo.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
Would define an helper `liqo.version` and use that for consistency. Additionally, it may also be a good idea to use the `required` construct to throw an error if neither `.Values.version` nor `.Chart.AppVersion`  is set, to avoid subtle errors (I didn't in CrownLabs and I deployed the chart multiple times without a valid tag for this reason). BTW, the AppVersion should be probably dropped from the chart and configured with `helm package`.

```
{{/*
The version of the application to be deployed
*/}}
{{- define "liqo.version" -}}
{{- .Values.version | default .Chart.AppVersion }}
{{- end }}
{{- end }}
```